pages/api/school/contact/create.js

```diff
@@ -0,0 +1,35 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  // If user is authenticated and is an admin
+  if (session && session.isAdmin) {
+    // Collect name of school
+    const { name: name, email: email, year: year, phone: phone, school_id: school_id } = req.body;
```

Comment:
No need to rename when destructuring.
```suggestion
    const { name, email, year, phone, school_id } = req.body;
```

Reply:
Do we need to do snake case to camel case conversion here? Had an instance of that above.